packages/itwinui-css/src/tile/tile.scss

```diff
@@ -169,6 +93,97 @@ $thumbnail-selector: ':is(.iui-thumbnail-icon, .iui-tile-thumbnail-picture)';
   //#endregion Statuses
 }
 
+@mixin iui-selected {
+  @include iui-focus($offset: 0, $thickness: var(--iui-size-2xs));
+  --_iui-tile-status-icon-fill: var(--iui-color-icon-accent);
+
+  outline: solid 2px var(--iui-color-border-accent);
+  background-color: var(--iui-color-background-accent-muted);
+}
+
+@mixin iui-actionable {
+  @include iui-focus($offset: var(--iui-size-3xs), $thickness: var(--iui-size-3xs));
+
+  &:has(.iui-link-action:focus-visible) {
+    outline: var(--iui-size-3xs) solid var(--iui-color-border-accent);
+
+    &.iui-selected {
+      outline-width: var(--iui-size-2xs);
+    }
+  }
+
+  @supports not selector(:has(+ *)) {
+    &:where(:focus-within) {
+      outline: var(--iui-size-3xs) solid var(--iui-color-border-accent);
```

Comment:
add the `iui-selected` portion in this fallback too. Right now in Firefox, there is no way to tell that a selected tile is focused

Reply:
Added iui-selected portion